include/superior_mysqlpp/low_level/mysql_hacks.hpp

```diff
@@ -75,6 +79,8 @@ namespace SuperiorMySqlpp { namespace LowLevel { namespace detail { namespace My
 {
     /*
      * Use this class only with thread_local storage specifier to ensure correct functionality!
+     *
+     * RAII class 
```

Comment:
Please add maybe short description or example, where and why it is used.

Reply:
Issue was filled, see #83 

Class comment was updated, please check updated PR.